INSTALL.md

```diff
@@ -30,53 +30,12 @@ Connect an Ethernet cable or set-up wifi. Power as normal via it's power input.
 
 ## Steps for all Pis
 
-Install chromium:
-
-    sudo apt-get update
-    sudo apt-get install rpi-chromium-mods
```

Comment:
I'm still unsure of what `rpi-chromium-mods` installs in addition to `chromium-browser`. It's worth remebering though in case we have any issues further down the line with our install of chromium.

Reply:
I've installed it after the manual installation of chromium and it seems happy enough, I'll add it to the script